pkg/extensions/sync/sync.go

```diff
@@ -63,6 +63,9 @@ type Remote interface {
 	GetRepoTags(repo string) ([]string, error)
 	// Get manifest content, mediaType, digest given an ImageReference
 	GetManifestContent(imageReference types.ImageReference) ([]byte, string, digest.Digest, error)
+	// In the case of public dockerhub images 'library' namespace is added to the repo names of images
+	// eg: alpine -> library/alpine
```

Comment:
Maybe the commit message should indicate this changeset is breaking an existing behavior, and people should update their configs?
Do we break it? Is it possible not to break it?

Reply:
We don't break it, it will still work as it is today, the only difference is that the bearer client would not work because docker.io/library is not the URL for docker APIs, it's a redirect to hub.docker.com.

So it will not send any WWW-Authenticate header for the client to use.

It will still sync images, but all calls to referrers (cosign, notation etc) will get html responses into the body, and it won't unmarshal the reponses..

Reply:
OK, makes sense. Maybe we should advise users on what endpoint to use. In one of the docs.